Library/Homebrew/cask/artifact/abstract_uninstall.rb

```diff
@@ -108,13 +108,21 @@ def uninstall_launchctl(*services, command: nil, **_)
         all_services.each do |service|
           ohai "Removing launchctl service #{service}"
           booleans.each do |with_sudo|
+            sudo_as_root = with_sudo
```

Comment:
Why do this instead of just doing `sudo_as_root: with_sudo` (or, ideally as said above, `sudo_as_root: sudo`)